basic-javascript/project-2.spec.js

```diff
@@ -1,3 +1,127 @@
 const funcs = require('./project-2');
 
-// whoops.. there is no test suite for this file. You'll simply just have to create one :/
+describe('getBiggest', () => {
+    test('6 or 7', () => {
+        const result = funcs.getBiggest(6,7);
+        expect(result).toBe(7)
+    });
+});
+
+describe('greeting', () => {
+    test('Norwegian', () => {
+        const result = funcs.greeting('Norwegian');
+        expect(result).toBe('Hello!');
+    });
+});
+
+describe('isTenOrFive', () => {
+    test('10is10', () => {
+        const result = funcs.isTenOrFive(10);
+        expect(result).toBe(true);
+    });
+});
+
+describe('isInRange', () => {
+    test('30IsInRange', () => {
+        const result = funcs.isInRange(30);
+        expect(result).toBe(true);
```

Comment:
Insufficient testing.. the implementation could just be:
```
function isInRange() {
    return true;
}
```
and your tests would accept the function